packages/maps/src/Client.ts

```diff
@@ -114,6 +114,16 @@ export interface AggregationColumn {
   aggregated_column: string;
 }
 
+export interface StatsColumn {
+  topCategories: number;
+  includeNulls: boolean;
+}
+
+export interface Sample {
+  num_rows: number;
+  include_columns: string[];
```

Comment:
Should we use camelCase for all interfaces (and map internally to snake_case if required)?

```
numRows: number;
includeColumns: string[];
```

Reply:
Agree, I just did the trick here 😄